src/components/icons/languages/index.ts

```diff
@@ -79,5 +80,6 @@ export const FLAVORS_ICONS = {
 	linux: LinuxIcon,
 	ruby: RubyIcon,
 	tailwind: TailwindIcon,
-	c: CIcon
+	c: CIcon,
+	figma: figmaIcon
```

Comment:
Replace `figmaIcon` to `FigmaIcon` PLs :D

Also Replace
`import { figmaIcon } from './figma'`
to this `import { figmaIcon } from './figma'`